webapp/src/types/github_types.ts

```diff
@@ -136,6 +136,6 @@ export type SidebarData = {
     yourPrs: GithubIssueData[];
     yourAssignments: GithubIssueData[],
     unreads: UnreadsData[]
-    org: string,
+    orgs: string,
```

Comment:
Is this supposed to be `string[]`?

Reply:
I'm wondering this didn't cause any type errors when it was `string` instead of `string[]`